### ISSUE TITLE
FAQ - Another H2 Headline

### ISSUE THREAD

**Issue by mcastro-lqd** [liqd/digitalstrategie#466]
**URL:** https://gd-dev.liqd.net/faq/
**Comment/Question:** *The "Why? How? Who? Superlong...?" ones should be H3s inside the H2 So many questions, right?* 

<img width="1056" alt="Screenshot 2022-05-23 at 16 58 50" src="https://user-images.githubusercontent.com/59610786/169848364-37e7771f-4f59-4e03-915a-55eff109eed2.png">

**Comment by phillimorland** [comment]
@mcastro-lqd yeah they should be h3 and they are h3 but this is due to content generated by the user, we are discussing this here https://github.com/liqd/digitalstrategie/issues/463 so will close this one
